subprojects/emcee/worker/src/test/kotlin/com/avito/emcee/worker/rest/stub/StubBucket.kt

```diff
@@ -0,0 +1,42 @@
+package com.avito.emcee.worker.rest.stub
+
+import com.avito.emcee.queue.ApkLocation
+import com.avito.emcee.queue.Bucket
+import com.avito.emcee.queue.BuildArtifacts
+import com.avito.emcee.queue.Payload
+import com.avito.emcee.queue.PayloadContainer
+import com.avito.emcee.queue.RemoteApk
+import com.avito.emcee.queue.TestConfiguration
+import com.avito.emcee.queue.TestExecutionBehavior
+
+fun Bucket.Companion.stub(bucketId: String) = Bucket(
```

Comment:
Personally Prefer to see fun `StubBucket`

Reply:
Answered about it above